compliance/compliance.go

```diff
@@ -1260,6 +1265,48 @@ func ReplaceMissingIPv4Entry(c *fluent.GRIBIClient, t testing.TB, _ ...TestOpt)
 			AsResult())
 }
 
+// TestOperationIsolation verifies no AFTOperation responses are received on a
+// second client after the primary client has disconnected.
+func TestOperationIsolation(c *fluent.GRIBIClient, t testing.TB, opts ...TestOpt) {
+	defer electionID.Add(2)
```

Comment:
why we want to +2? Next func can simply `.Load()+1` if they want to ensure to be the primary, right?

Reply:
Each test case expects the value of electionID.Load() to be a good value to use for leader election (i.e., [here](https://github.com/openconfig/gribigo/blob/71d18fce9c447b64639f4f81955dffdac7fd41d9/compliance/election.go#L97)).  This test case consumes two election IDs (one per client), so it needs to be incremented by 2 for the next test case to get the highest leader id.